### PR TITLE
Add minimal README to docs directory

### PR DESCRIPTION
**Pull request by axefrog**
Just a convenient link to the readthedocs site.

### CODE REVIEW

**Comment by briancavalier** [comment]
It looks like there are some extra commits in the PR.  Could you rebase on latest master so it just has the one relevant commit?  Thanks!

**Comment by axefrog** [comment]
I've never rebased anything before. How do you rebase something that is already on a master branch?

**Comment by TylorS** [comment]
In your clone you want to do something like this at a shell of some kind 
```
git remote add $name https://github.com/mostjs/core

git rebase $name/master
```
Where `$name` can be whatever you want.

**Comment by axefrog** [comment]
Ok, so I did that and got this; not sure how to interpret it though:

```
D:\dev\most [master ≡]> git rebase source/master
First, rewinding head to replay your work on top of it...
Applying: fix(types): propagateTask; decouple value type from sink type
Using index info to reconstruct a base tree...
M       packages/core/type-definitions/PropagateTask.d.ts
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
Applying: fix(types): propagateTask; decouple value type from sink type (flow)
Using index info to reconstruct a base tree...
M       packages/core/src/index.js.flow
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.
Applying: docs(api): Add minimal README to docs directory
```

When I do git status, it says:

```
D:\dev\most [master ↓4 ↑1]> git status
On branch master
Your branch and 'origin/master' have diverged,
and have 1 and 4 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
nothing to commit, working tree clean
```

**Comment by TylorS** [comment]
`rebase` changes the git history. After doing a rebase it's often you have to do a force push. `git push - f` because your local branch and remote branches are out of sync

**Comment by briancavalier** [comment]
> How do you rebase something that is already on a master branch?

It looks like what happened was that your fork's remote master may not have been up-to-date with the upstream (the mostjs/core repo) when the PR was opened.  Hard to tell for sure, though, since there's also a merge commit (a059130) after the change to the README ... not clear where that came from.

In any event, thanks for giving it a go.  If it doesn't end up looking right after the rebase and force push, we have a couple other options.  For example, either you or one of us can create a new branch from up-to-date master and cherry-pick the one relevant commit (0b1434d) to it, and open a new PR.

**Comment by axefrog** [comment]
Commit log should be fixed now.

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Almost there...
+
+Documentation for Most.js can be found at [mostcore.readthedocs.io](http://mostcore.readthedocs.io).
```
Comment (briancavalier) [inline docs/README.md]: Couple small requests: Let's change "Most.js" to "`@most/core`", and make the link `https`.  Thanks!

Comment (axefrog) [reply docs/README.md]: Done.

**Pull request merged**